apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js

```diff
@@ -841,4 +857,18 @@ describe('CircuitPlaygroundBoard', () => {
       }
     });
   });
+
+  /**
+   * Simulate a raw value coming back from the board on the given component's pin.
+   * @param {five.Sensor|five.Thermometer} component
+   * @param {number} rawValue - usually in range 0-1023.
+   * @throws if nothing is monitoring the given analog pin
+   */
+  function setSensorAnalogValue(component, rawValue) {
```

Comment:
Is there somewhere shared we could put this helper function (this is the same as what's in playgroundComponentsTest, yeah)?

Also wondering if we could create a helper in the same place for the stubbing you're doing that takes in the component you're stubbing as an argument? Relatedly, is it intentional to only callThrough() on the thermometer, but not the sensor?